src/auth/choicewindow.cpp

```diff
@@ -0,0 +1,89 @@
+#include "choicewindow.h"
+#include "downloaderWrapper.h"
+
+#include <QLabel>
+#include <QListWidget>
+#include <QVBoxLayout>
+#include <QDialogButtonBox>
+#include <QAbstractButton>
+#include <QPushButton>
+#include <QDebug>
+#include <QApplication>
+#include <QDesktopWidget>
+
+ChoiceWindow::ChoiceWindow(QStringList &usersFilesList, QWidget *pParent)
+    : QDialog (pParent)
+{
+    window()->setFixedSize( window()->sizeHint() );
+    setModal(true);
```

Comment:
1) What is the purpose of calling window()? You might just call setFixedSize()
2) window()->sizeHint() doesn't work. It returns QSize(-1, -1).